aksparse/server/Aksparse.chpl

```diff
@@ -232,6 +232,7 @@ module Aksparse {
                 var cumsum_symentry = toSymEntry(gEntCumSum,int);
                 var cumsum_arr = cumsum_symentry.a;
 
+                // RIGHT HERE?????
```

Comment:
Is this question answered? I don't see any immediate issues, so if there is nothing in question here, we should remove this. If it is still needed I would recommend updating to a more descriptive `TODO` comment.

Reply:
Thanks for catching this. Left a note for myself to consider if this code was reusable, then forgot about it. Removed